pkg/pillar/hypervisor/xen.go

```diff
@@ -456,7 +456,9 @@ func (ctx xenContext) Info(domainName string, domainID int) (int, types.SwState,
 
 	status, err := ioutil.ReadFile("/run/tasks/" + domainName)
 	if err != nil {
-		log.Errorf("couldn't read task status file: %v", err)
+		err = fmt.Errorf("Info(%s, %d): couldn't read task status file: %v", domainName, domainID, err)
```

Comment:
I don't understand this part -- the file is guaranteed to be there simply based on the sequencing of events in domaingr and xen-start script. Please get to the bottom of what really is going on -- I don't think we can merge this workaround.

Reply:
Removed.